packages/frontend/src/state/crab/hooks.ts

```diff
@@ -497,6 +497,11 @@ export const useFlashDepositV2 = (calculateETHtoBorrowFromUniswap: any) => {
       if (!contract || !vault) return
 
       let { ethBorrow: _ethBorrow } = await calculateETHtoBorrowFromUniswap(amount, slippage)
+      // Just to make sure the issue never happens
+      if (_ethBorrow.isZero()) {
+        alert('Some error occurred. Refresh the page!')
+        throw new Error('Some error occurred. Refresh the page!')
```

Comment:
Do we also want to disable the button in this case? Otherwise imo this works as an interim quick soln until the fix is complete